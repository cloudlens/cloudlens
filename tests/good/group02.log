foo 1
foo 1
foo 3
foo 3
foo 5
foo 5
foo 7
foo 7
foo 9
foo 9
