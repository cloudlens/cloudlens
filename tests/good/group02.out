Before group: e = {"message":"foo 1","n":"1"}
Before group: e = {"message":"foo 1","n":"1"}
Before group: e = {"message":"foo 3","n":"3"}
Before group: e = {"message":"foo 3","n":"3"}
Before group: e = {"message":"foo 5","n":"5"}
Before group: e = {"message":"foo 5","n":"5"}
Before group: e = {"message":"foo 7","n":"7"}
Before group: e = {"message":"foo 7","n":"7"}
Before group: e = {"message":"foo 9","n":"9"}
Before group: e = {"message":"foo 9","n":"9"}
cpt = 0
